finances.js

```diff
@@ -198,8 +204,7 @@ function deleteTransaction(trID) {
         renderTransactions(transactions);
 
         document.getElementById("transaction-form").reset();
```

Comment:
the form reset will clear out the fields and you can't enter data again until you fill in the required fields - we just need to maintain the state by updating the button text.